### PR TITLE
added some packages to the source folder

### PR DESCRIPTION
**Pull request by khannoori**

### CODE REVIEW

File: src/helpers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+'''
```
Comment (kanafghan) [inline src/helpers/__init__.py:1]: This code does not belong to this project. Please remove these.

**Comment by kanafghan** [comment]
Please remove the following packages and files since we do not need them:
- `src/api/external`
- `src/api/internal`
- `src/api/importers`
- `src/api/lib`
- `src/helpers/engines.py`
- `src/helpers/ocr.py`
- `src/helpers/servers.py`
- `src/helpers/uploadhandler.py`
- `src/helpers/uploadhandler.py`

**Comment by toin0u** [comment]
@kanafghan I think you need to delete this PR because of the thing we discussed the other day ;)

**Comment by kanafghan** [comment]
@toin0u Yes you are right. I will delete it if possible.

@khannoori I will tell you I deleted this and how you can create another PR with the proper changes.

**Comment by khannoori** [comment]
Modtaget :+1:

**Pull request closed without merging**